CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ This version has breaking changes, `Decidim::Feature` has been renamed to `Decid
 and also everything related to it (controllers, views, etc.). If you have customised some
 controller or added a new module you need to rename `feature` to `component`.
 
+With the addition of the new step "Complete" to the proposal creation wizard,
```

Comment:
Will this be required or just recommended?

Reply:
Recommended for those admins that filled in the `step_3_help_text`.
With 3 steps, the `step_3_help_text` is for the _Preview_ and last step.
So adding the new step _Complete_ before the _Preview_, the `step_3_help_text` is for the _Complete_ step and not the _Preview_ .

Reply:
I guess that's the problem with enumerating steps instead of giving them a name 😞